solutions/instances/outputs.tf

```diff
@@ -2,6 +2,22 @@
 # Outputs
 ##############################################################################
 
+## Cloud logs
+output "crn" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_crn : null
+  description = "The id of the provisioned Cloud Logs instance."
+}
+
+output "guid" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_guid : null
+  description = "The guid of the provisioned Cloud Logs instance."
+}
+
+output "name" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_name : null
+  description = "The name of the provisioned Cloud Logs instance."
+}
```

Comment:
Seems like its duplicated. You are doing it twice.
Also, var name should not e only `crn` / `guid` etc...

Reply:
Removed the duplicated code and updated the output names.